charms/reactive/relations.py

```diff
@@ -48,19 +49,20 @@ def relation_from_name(relation_name):
         return factory.from_name(relation_name)
 
 
-def relation_from_state(state):
-    """The object used for interacting with relations tied to a state, or None.
+@deprecated.alias('relation_from_state')
+def relation_from_flag(flag):
+    """The object used for interacting with relations tied to a flag, or None.
```

Comment:
Alas we need the alias here, because of people being clever with relation_from_state. Its good we are flagging things internal only now.